lib/zstd.h

```diff
@@ -1834,6 +1841,19 @@ ZSTDLIB_API size_t ZSTD_CCtx_refPrefix_advanced(ZSTD_CCtx* cctx, const void* pre
  */
 #define ZSTD_c_validateSequences ZSTD_c_experimentalParam12
 
+/* ZSTD_c_useRowMatchfinder
+ * Default is 0 == ZSTD_urm_auto.
+ * Controlled with ZSTD_useRowMatchfinderMode_e enum.
+ * 
+ * By default, in ZSTD_urm_auto, when finalizing the compression parameters, the library
+ * will decide at runtime whether to use the new row-based matchfinder. If so, the final
+ * applied compression parameters will change to suit the new row-based matchfinder.
+ * 
+ * Set to 1 = ZSTD_urm_disableRowMatchfinder to never use row-based matchfinder.
+ * Set to 2 = ZSTD_urm_enableRowMatchfinder to force usage of row-based matchfinder.
```

Comment:
I wouldn't specify the enum values here, since that is an implementation detail. We don't want people passing raw integers.